modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/impl/MetaStorageManagerImpl.java

```diff
@@ -855,19 +824,19 @@ private void onSafeTimeAdvanced(HybridTimestamp time) {
     }
 
     /**
-     * Saves processed Meta Storage revision and corresponding entries to the Vault.
+     * Saves processed Meta Storage revision to the {@link #appliedRevision}.
      */
-    private CompletableFuture<Void> onRevisionApplied(WatchEvent watchEvent) {
+    private CompletableFuture<Void> onRevisionApplied(long revision) {
```

Comment:
1. This method doesn't need to return a future anymore.
2. All it does is assigning a field, I guess we no longer need the busy lock as well.